packages/@react-spectrum/actiongroup/docs/ActionGroup.mdx

```diff
@@ -0,0 +1,211 @@
+  packageData={packageData}
+  componentNames={['ActionGroup']} />
+
+## Example
+
+```tsx example
+<ActionGroup>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+## Content
+
+ActionGroups accept as list of `children`, each with a `uniqueKey` prop. Generally `children` will be `Item` components from the `@react-stately/collections` library.
+
+However, a function that returns `Item` components is also accepted.
+
+```tsx example
+const items = [
+  {children: 'React', name: 'React'},
+  {children: 'Add', name: 'Add'},
+  {children: 'Delete', name: 'Delete'}
+];
+<ActionGroup items={items} itemKey="name">
+  {item => <Item textValue={item.name}>{item.children}</Item>}
+</ActionGroup>
+```
+
+
+In order to identify children when a user takes an action on a item, each child needs to have a `uniqueKey` prop. If using a function, an `itemKey` prop is required on the ActionGroup.
+
+See [Events](#events) for more information.
+
+## Value
+
+ActionGroups are uncontrolled by default. 
+
+Use `defaultSelectedKeys` to provide a default set of selected items. The value of the selected keys must match the `uniqueKey` prop of the items.
+
+```tsx example
+<ActionGroup defaultSelectedKeys={['delete']}>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+The `selectedKeys` prop can be used to make the selected state controlled.
+
+```tsx example
+<ActionGroup selectedKeys={['delete']}>
+  <Item uniqueKey="add">Add</Item>
+  <Item uniqueKey="delete">Delete</Item>
+  <Item uniqueKey="edit">Edit</Item>
+</ActionGroup>
+```
+
+### Accessibility
+
+If no visible label is provided (e.g. an icon only) to each item, an alternative text label must be provided to identify the control for accessibility.
+This should be added using the `aria-label` prop to each Item.
+
+```tsx example
+<ActionGroup>
+  <Item uniqueKey="brush" aria-label="Brush"><Brush /></Item>
+  <Item uniqueKey="select" aria-label="Select"><Select /></Item>
+  <Item uniqueKey="regionSelect" aria-label="Select Region"><RegionSelect /></Item>
+</ActionGroup>
+```
+
+### Internationalization
+
+In order to internationalize an ActionGroup, the strings of all child items should be localized.
+
+## Events
+
+Use the `onSelectionChange` prop as a callback to handle press events on items.
+The `uniqueKey` prop from the selected item will be passed into the callback.
+
+The following example uses the `onSelectionChange` prop to update a string stored in React state.
+
+```tsx example
+function Example() {
+  let [lastKey, setLastKey] = React.useState(new Set());
+  return (
+    <div>
+      <ActionGroup onSelectionChange={setLastKey}>
+        <Item uniqueKey="add">Add</Item>
+        <Item uniqueKey="delete">Delete</Item>
+        <Item uniqueKey="edit">Edit</Item>
+      </ActionGroup>
+      <p>Last selected: {lastKey}</p>
```

Comment:
isn't this current selection? it stays in the selected state after blur

Reply:
good catch!